doc/use_cases/navigation_with_cartographer.md

```diff
@@ -0,0 +1,44 @@
+
+**Navigation 2 with Cartogtographer**
+
+This document explains how to use Navigation 2 with Cartographer. The following steps show ROS 2 users how to generate occupancy grid maps and use Navigation 2 to move their robot around.
+
+## Before the Tutorial
+
+- Install ROS 2
+- Install Navigation 2
+- Install Cartographer
+- Install your robot packages. 
+- Source setup.bash in your ROS2 and Navigation 2 installation/build directory or workspaces
+- Setting the ROS Domain ID variable is recommended. Ex:
```

Comment:
I might remove the ROS domain ID. If people need some non-default value they'll know to set it but a typical user probably doesnt need to know / doesnt do anything special for this application